src/util/duration.cpp

```diff
@@ -3,7 +3,7 @@
 #include <QtGlobal>
 #include <QStringBuilder>
 #include <QTime>
-
+#include <math.h>
```

Comment:
I'm a bit confused by this addition. It has no path and the syntax doesn't match the other entries. Besides which, two lines below is the statement `#include "util/math.h"` - is that a different file?

Reply:
`"util/math.h"` (in mixxx's codebase) is preferable to `<math.h>` (the [C standard library header](http://www.cplusplus.com/reference/cmath/)), since it includes Mixxx-specific helpers / shims.